packages/api/src/external/carequality/command/cq-directory/get-organizations-for-xcpd.ts

```diff
@@ -14,6 +15,16 @@ function sortOrgsByNearbyOrder(orgs: CQDirectoryEntryModel[], orderMap: Map<stri
   });
 }
 
+export async function getAllCQOrgsIds(): Promise<Set<string>> {
```

Comment:
only getting the Ids of organizations with a valid XCPD link. Using a set so we can have O(1) lookups instead of filtering through a list